asciidoc/sdpi-supplement-intro.adoc

```diff
@@ -6,11 +6,14 @@
 [%autowidth]
 [cols="1"]
 |===
-a| *SDPi 1.2 Supplement -- _STU / TI Version_ -- Note*:
+a| *SDPi 1.3 Supplement -- _STU / TI Version_ -- Note*:
 
-This release of the SDPi 1.2 Standard for Trial Use (HL7 STU) / Trial Implementation (IHE TI) supplement is the first release that is intended for both an HL7 ballot cycle (January 2024), as well as the same IHE Devices release and review process, which was used for the first 3 releases of this supplement.
-Release 1.2 provides only a few _incremental_ updates to release 1.1; however, it continues the objective of quarterly _minor version_ releases and continues to advance support for prototyping and testing of device-to-device plug-and-trust interoperability.
-It is recognized, though, that this release is incomplete with work continuing for 1.3 and subsequent versions.
+This release of the SDPi 1.3 Standard for Trial Use (HL7 STU) / Trial Implementation (IHE TI) supplement is the first release that integrates resolutions from an HL7 ballot cycle (January 2024). This process implements links from HL7 ballot Jira tickets to IHE DEV.SDPi Github Issues that then enable tracking of the comment resolution implementation in this specification.
+
+Note:  The 1.3.1 release provides a few minor editorial corrections that were overlooked in the first release of 1.3.0.
+
+Release 1.3 provides only a few _incremental_ updates to release 1.3; however, it continues the objective of quarterly _minor version_ releases and continues to advance support for prototyping and testing of device-to-device plug-and-trust interoperability.
```

Comment:
phrase should read "...updates to release 1.2"

Reply:
Grrrrrr ...

Reply:
Is that a Grrrr of agreement or disagreement? ;-)